src/content/docs/billing.mdx

```diff
@@ -0,0 +1,35 @@
+---
+title: Billing
+description: How Shorebird handles Billing
+---
+
+Shorebird uses Stripe for all billing. We accept payment in any method
```

Comment:
Seems very closely related to pricing. I wonder if we shouldn't at least link to billing from pricing/vice-versa or even consider consolidating this into the pricing page 🤷

Reply:
I agree, they are related.

Reply:
We can clean them up further.  Landing this for now.